variables/_layers.pcss

```diff
@@ -0,0 +1,7 @@
+:root {
+	/* -----------------------------------------------------------------------------
+	 * Layers of z-index
+	 * ----------------------------------------------------------------------------- */
+
+	--layer-spinner-container: 100;
```

Comment:
also maybe call this `--z-index-spinner-container` if we're calling this file `_z-index.pcss`.